package.json

```diff
@@ -19,14 +19,13 @@
     ],
     "icon": "images/icon.png",
     "engines": {
-        "vscode": "^1.46.0"
+        "vscode": "^1.76.0"
     },
     "categories": [
         "Other"
     ],
     "activationEvents": [
-        "onStartupFinished",
-        "onCommand:quick-file-browser.open"
```

Comment:
Removed this because doesn't seem necessary anymore and the linter was complaining:
```
This activation event can be removed as VS Code generates these automatically from your package.json contribution declarations.
```